### PR TITLE
Add Alive nodes limit row, show maximum number of alive nodes

### PR DESCRIPTION
**Pull request by tobwiens**
Add Alive nodes limit row

A alive node limit row is added. It shows the corresponding maxNumberOfNodes. If the alive nodes are not limited, the description is changed to alive nodes unlimited and the corresponding maxNumberOfNodes is displayed next to it, which is a negative number.

### CODE REVIEW

**Comment by tobwiens** [comment]
Nodes limited: 



Nodes unlimited:

**Comment by tobwiens** [comment]
@ow2-proactive/developers two reviews please.

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java [hunk 1]
```diff
@@ -325,6 +325,36 @@ public void run() {
             }
         };
         this.statsUpdater.scheduleRepeating(RMConfig.get().getStatisticsRefreshTime());
+
+        scheduleRequestMaxNumberOfNodesIn(0);
+        }
+
+    private void scheduleRequestMaxNumberOfNodesIn(int milliseconds) {
+        new Timer() {
```
Comment (lpellegr) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java]: Why do you need a Timer here, especially with value `0`? is the alive nodes limit not static?

File: rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java [hunk 2]
```diff
@@ -325,6 +325,36 @@ public void run() {
...
+                fetchNodesLimit();
+            }
+        }.schedule(milliseconds);
+    }
+
+    private void fetchNodesLimit() {
+        this.rm.getState(LoginModel.getInstance().getSessionId(), new AsyncCallback<String>() {
+            public void onSuccess(String result) {
+                // Parse json response to extract the current node limit
+                LogModel.getInstance().logMessage("getState response:"+result);
```
Comment (lpellegr) [inline rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java]: It looks like our formatting rules are not applied?

**Pull request closed without merging**